pyanaconda/bootloader/execution.py

```diff
@@ -63,6 +63,9 @@ def execute(self, storage, dry_run=False):
             log.debug("Bootloader is not enabled, skipping.")
             return
 
+        # Update the disk list.
+        storage.bootloader.set_disk_list([d for d in storage.disks if d.partitioned])
```

Comment:
Please add the information that the list is already sorted. Right now it is not obvious from this code.
Maybe the best place would be in the `storage.disks` method documentation. If it is not there already.

Reply:
The property `disks` belongs to Blivet. I have added a comment here.